services/sts/systests/sts-itests/src/test/java/org/apache/cxf/systest/sts/itests/BasicSTSIntegrationTest.java

```diff
@@ -59,10 +60,10 @@ public Option[] getConfig() {
             systemProperty("BasicSTSIntegrationTest.PORT").value(port),
             editConfigurationFilePut("etc/org.ops4j.pax.web.cfg",
                                     "org.osgi.service.http.port", port),
-            when(!localRepository.isEmpty())
-                .useOptions(systemProperty("org.ops4j.pax.url.mvn.localRepository").value(localRepository)),
+            when(!localRepository.isEmpty()).useOptions(editConfigurationFilePut("etc/org.ops4j.pax.url.mvn.cfg",
```

Comment:
For some reasons, the `systemProperty` is not working but setting it in `etc/org.ops4j.pax.url.mvn.cfg` does, I will bring this change to separately (if you don't mind).

Reply:
Please go ahead

Reply:
@reta is there something you need me to do here like rebase the PR perhaps?

Reply:
@rotty3000 my apologies, a bit behind pushing the change to master, but yes, would be great if you could rebase after, thank you (hope to be get it in with next few hours).

Reply:
@rotty3000 change is in master, https://github.com/apache/cxf/pull/948, could you rebase please? thank you!

Reply:
rebase done! :)